src/components/AdjustLeveragePosition/AdjustLeveragePosition.tsx

```diff
@@ -182,7 +241,7 @@ const AdjustLeveragePosition: FC<AdjustPositionProps> = ({
       return null;
     }
 
-    return collateralAmountDecimal.mul(Decimal.ONE.div(rate));
+    return collateralAmountDecimal.div(rate);
```

Comment:
We are converting selected collateral token amount (stETH or wstETH) to (wstETH). Rate that we get from rate map is wstETH -> stETH. So that's why we have to flip this rate to get correct one (stETH -> wstETH).